lib/news_scraper/transformers/article.rb

```diff
@@ -20,22 +19,26 @@ def transform
         transformed_response.merge(root_domain: root_domain)
       end
 
+      def self.scrape_patterns
```

Comment:
Right, but if we ever want to use this elsewhere besides the `Trainer`, e.g. `NewsScraper.scrape('shopify')`

```
module NewsScraper
   extend self
   
   def scrape(query)
      links = Extractors::GoogleNewsRss.new(query: query).extract
      links.each do |link| 
         raw_html = Extractors::Article.new(query: query).extract
         transformed_data = Transformers::Article.new(uri: link, paylaod: raw_html).transform # This will reload the static YAML file (since it's not being trained) every time in initialize
      end
   end
end
```